lib/summary.rb

```diff
@@ -0,0 +1,27 @@
+class Summary
+
+  def initialize(order)
+    @order = order
+    @count = 0
+  end
+
+  def run
+    total
+    retro
+  end
+
+  private
+
+  def total
+    @order.each do |item|
+      puts item
+      item.each_value do |price|
+        @count += price
+      end
+    end
+  end
+
+  def retro
+    return "You've ordered #{@order} and it cost £#{@count}"
```

Comment:
i cant seem to see what '@order' is returning? 
is this an array of dishes? order number? something else? 
I really like this feature, great to provide the user with info, is there any other info that you have to hand as instance variables or likewise in that the user might want displayed at this point? (order status, etc)

Reply:
the @order is returning what the user have selected to order so it will be an array of hashes, where he will be able to see the name of his dish and its individual price (if he wants to recalculate the price himself to be sure)